container.fc

```diff
@@ -12,15 +12,14 @@
 /usr/bin/docker-novolume-plugin	--	gen_context(system_u:object_r:container_auth_exec_t,s0)
 /usr/sbin/crio.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /usr/bin/crio.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
-/usr/bin/ocid.*			--	gen_context(system_u:object_r:container_runtime_exec_t,s0)
```

Comment:
I don't think we should remove these until we release cri-o.  So leave these in for now.